doc/grids.rst

```diff
@@ -324,10 +324,10 @@ For example:
 
 .. ipython:: python
 
-    f2 = f + xr.Dataset(coords={"y": np.arange(1, 3)})["y"]
-    f2 = f2.assign_coords(h=f2.y ** 2)
-    print(f2)
-    grid.interp(f2, "X", keep_coords=True)
+    da2 = da + xr.Dataset(coords={"y": np.arange(1, 3)})["y"]
+    da2 = da2.assign_coords(h=da2.y ** 2)
+    print(da2)
+    grid.interp(da2, "X", keep_coords=True)
```

Comment:
I changed this because I reuse the same example data in the grid ufunc docs, and I thought it would be confusing to have both a dataarray python variable called `f`, but also be passing functions as arguments to other functions.